src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -303,6 +303,25 @@ protected function seeJsonContains(array $data, $negate = false)
     }
 
     /**
+     * Assert that the response is a superset of the given JSON.
+     *
+     * @param  array  $data
+     * @return $this
+     */
+    protected function seeJsonSubset(array $data)
+    {
+        $actual = json_decode($this->response->getContent(), true);
+
+        if (is_null($actual) || $actual === false) {
+            return $this->fail('Invalid JSON was returned from the route. Perhaps an exception was thrown?');
+        }
```

Comment:
This is unfortunately duplicated from `seeJsonContains`, which is where the rest of the JSON assertions flow through. 

How would you guys like to prevent this duplication? I don't *think* returning `$this->fail` is actually important (I think the fail happens immediately) so we could likely just extract this into a method for validating the JSON response.